src/js/module/dashboard/ListIssuesWAQ.js

```diff
@@ -84,6 +100,15 @@ class ListIssuesWAQ extends React.Component {
         return (
             <div className="panel mb-3">
                 <Title text={`WAQ ${issueCount ? `(${issueCount})` : ''}`} />
+                <form className="form-inline">
```

Comment:
I think it might look nicer if we could make this checkbox part of the panel title. That might mean not using the `<Title>` component directly. Could you try this please?